functional_tests/scylla_operator/test_functional.py

```diff
@@ -12,7 +12,10 @@
 # See LICENSE for more details.
 #
 # Copyright (c) 2021 ScyllaDB
+
+#  pylint: disable=too-many-lines
```

Comment:
maybe it's good sign to start breaking it into more files...

Reply:
Probably. 
But it requires decision about the criteria for splitting.
First thought that comes into my mind is following:
- common (majority of tests)
- mgmt
- cloud-bundle -specific

This new test is not really different than the existing `test_deploy_helm_with_default_values` one.

Reply:
No, we should split it to more files:
- sanity/basic
- multiple clusters
- reproducers

in theory we can have one file per test even... we don't have they many

Reply:
and yes it's not blocking